draft-zsf.md

```diff
@@ -0,0 +1,168 @@
+```
+ZIP: 
+Title: Establish the Zcash Sustainability Fund on the Protocol Level
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits:
+Status: Draft
+Category: Ecosystem
+Created: 2023-08-
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED", "OPTIONAL", and "REQUIRED" in this document are to be interpreted as described in RFC 2119. [1]
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200. [2]
+
+The term "Block Rewards" refers to the algorithmic
+issuance of ZEC to every block's creator -- part of the consensus rules.
```

Comment:
This is slightly incorrect and should be replaced with the correct definition from: https://github.com/zcash/zips/pull/706/files#r1333620406

Reply:
Fixed, I will update the definitions to reference these in the other PRs